cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -669,7 +669,7 @@ def is_email_to_send_run_completion(self, status):
     def is_email_to_send_callback(self, status, time_original_request):
         if not self.public:
             # in case the request was long and 'done'
-            if status == 'done':
+            if status in ['done', 'task_complete']:
```

Comment:
Is `task_complete` a status that can be returned? I ask just because I have never seen 'til now. Does it have the same meaning as `done`?

Reply:
There are many statuses meaning different things. Particular choice will end up in different email rate. So maybe it's best configurable too. Later.

Reply:
I was about indeed to suggest a configurable approach